config/locales/views/rents/rents.sv.yml

```diff
@@ -1,62 +1,57 @@
+      cannot_edit_rent: "Du kan inte längre redigera bokningen"
       council_booking: Utskottsbokning
-      edit_user_information: Redigera användaruppgifter
-      only_for_council: får endast göras åt utskottet
-      overlaps_with: 'Överlappar med:'
+      edit_user_information: "Redigera användaruppgifter"
+      only_for_council: "får endast göras åt utskottet"
+      overlaps_with: "Överlappar med:"
       terms: Bokningsavtal
-      the_terms: bokningsavtalet.
-      read_terms: Läs igenom och godkänn
-      agree_to_terms: Jag godkänner bokningsavtalet
-
+    car_bans:
+      banned: "Du är spärrad från bilbokningstjänsten."
+      reason: "Anledning: "
+      for_councils: "Utskottsbokning kan fortfarande genomföras."
     index:
-      ask_question: Ställ ny fråga
-      description: Sektionen har en bil till medlemmars förfogande, bokning sker via denna sida.
+      ask_question: "Ställ ny fråga"
+      description: "Sektionen har en bil till medlemmars förfogande, bokning sker via denna sida."
       keywords:
         - bil
         - flytta
-        - köra bil
-      new_rent: Ny bokning
+        - "köra bil"
+      new_rent: "Ny bokning"
       overview: Översikt
       terms: Bokningsavtal
       title: Bilbokning
-
     rent:
       title: Bilbokning
-
     renter_table:
-      user_field_with_asterix: Fält markerade med * måste ifyllas i användarprofilen
-
+      user_field_with_asterix: "Fält markerade med * måste ifyllas i användarprofilen"
     show:
-      administrate: Administrera bokning
-      all_rents: Alla bilbokningar
-      edit_rent: Redigera bokning
+      administrate: "Administrera bokning"
+      all_rents: "Alla bilbokningar"
+      edit_rent: "Redigera bokning"
       title: Bilbokning
-
     edit:
-      confirm_destroy: Din bokning kommer nu att tas bort
-      destroy: Förinta bilbokning
-      index: Startsida för bilbokning
-      overview: Dina bokningar
-      title: Redigera bilbokning
-
+      confirm_destroy: "Din bokning kommer nu att tas bort"
+      destroy: "Förinta bilbokning"
+      index: "Startsida för bilbokning"
+      overview: "Dina bokningar"
+      title: "Redigera bilbokning"
     overview:
-      title: Bilbokning - Översikt
-      new_rent: Ny bilbokning
-      index: Startsida för bilbokning
-      contact_forman: Kontakta bilförman
+      title: "Bilbokning - Översikt"
+      new_rent: "Ny bilbokning"
+      index: "Startsida för bilbokning"
+      contact_forman: "Kontakta bilförman"
       terms: Bokningsavtal
-      your_rents: Dina bokningar
-      edit_rent: Redigera bilbokning
-
+      your_rents: "Dina bokningar"
+      edit_rent: "Redigera bilbokning"
     new:
-      index: Startsida för bilbokning
-      title: Ny bilbokning
-
-    car_bans:
-      banned: Du är spärrad från bilbokningstjänsten.
-      reason: "Anledning: "
-      for_councils: Utskottsbokning kan fortfarande genomföras.
+      index: "Startsida för bilbokning"
+      title: "Ny bilbokning"
```

Comment:
I would like you to follow the new translation workflow, namely to not use the Swedish translation file exported via OneSky. Since you already have added everything to OneSky you only need to reset `rents.sv.yml` to the current `master` and then simply add the new Swedish phrases to it manually.